src/mesh/interpolation.cxx

```diff
@@ -178,10 +186,11 @@ const Field3D interp_to(const Field3D &var, CELL_LOC loc)
 	  }
 	}
 	break;
-	// Need to communicate in Y
       }
       case CELL_ZLOW: {
-        for(const auto &i : result.region(RGN_NOZ)) {
+        REGION this_region = region;
+        if ( region==RGN_ALL ) this_region=RGN_NOZ;
```

Comment:
This looks a little suspicious -- we're overriding an input argument for one specific case. I think this needs a comment to explain why there's special handling here.

Reply:
This might well just be wrong. I meant to keep the existing behaviour with RGN_NOZ in the 'default' case. Looking again I think I'd favour just passing the region argument (removing the special handling).

This can't ever have been used, because until this PR result.region(RGN_NOZ) would have thrown an exception as it was not implemented in Field3D. So I think it's safe to change the previous behaviour.